app/helpers/application_helper.rb

```diff
@@ -0,0 +1,16 @@
+module ApplicationHelper
+  def bootstrap_class_for flash_type
+    case flash_type
+      when "success"
```

Comment:
`when` should have the same indent as `case`.  Check out this [style guide](https://github.com/bbatsov/ruby-style-guide).